src/component/BreadCrumb.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {Breadcrumbs, Typography} from "@mui/material";
+
+function BreadCrumb({data}) {
+  const values = Object.values(data);
+  return (
+    <div>
+      <Breadcrumbs
+        aria-label="breadcrumbs"
+        sx={{
+          Weight: 400,
+          fontSize: "1.5rem",
+          Size: "1.5rem",
+          lineHeight: "2.25rem",
+          Letter: "0.25 px",
+        }}
+      >
+        {[...values].map((item) => (
```

Comment:
`values` is an array already, we dont need to spread and wrap it in array. for saftey check we can check if it is an array 
```
{ values.length && values.map((item) => (
...
```

Reply:
removed spreading of array value